climpred/predictability_horizon.py

```diff
@@ -0,0 +1,60 @@
+import numpy as np
+import xarray as xr
+
+
+def _last_item_cond_true(cond, dim, shift=-1):
+    """Get the last item of condition ``cond`` which is True.
+
+    Args:
+        cond (xr.DataArray, xr.Dataset): Condition True means predictable. ``cond``
+            contains at least the dimension lead and checks for how many leads the
+            ``condition`` is true and hence skill is predictable.
+        dim (str): Dimension to check for condition == True over.
+        shift (int): Description of parameter `shift`. Defaults to 1.
```

Comment:
Is this needed? Since this is a `_` func and only used here and `shift` isn't called I imagine you can get rid of the keyword since it's 0-based indexing and will always be `-1`.